wolfssl/wolfcrypt/port/Renesas/renesas-tsip-crypt.h

```diff
@@ -202,15 +108,305 @@ typedef wolfssl_TSIP_Hash wc_Sha;
     typedef struct {
         tsip_aes_key_index_t tsip_keyIdx;
         word32               keySize;
+        byte                 setup;
     } TSIP_AES_CTX;
+#endif 
+
+
+
+
+/****
+typedef struct user_PKCbInfo {
```

Comment:
Should this comment be deleted, about `user_PKCbInfo`?

Reply:
Deleted the commented out portion.